src/services/user.ts

```diff
@@ -47,7 +49,17 @@ export async function fetchById(userId: number): Promise<User> {
  */
 export async function save(user: any) {
   try {
-    const [id] = await userModel.save(user);
+    const codeLength = 6;
+
+    const mobileCode = randomCode(codeLength);
+    const emailCode = randomCode(codeLength);
+
+    const newUser = {
+      ...user,
+      emailVerificationCode: emailCode,
+      mobileVerificationCode: mobileCode
```

Comment:
We could even do this instead of what we are doing right now.

```suggestion
    const mobileVerificationCode = randomCode(codeLength);
    const emailVerificationCode = randomCode(codeLength);

    const newUser = {
      ...user,
      emailVerificationCode,
      mobileVerificationCode
    };
```

Reply:
done